package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "@justeat/f-recruit-message",
   "description": "",
-  "version": "0.0.5",
+  "version": "0.0.6",
```

Comment:
Without being pedantic – this aint a patch release ;)

Should be a v0.1 release